File: Python/libraries/datatypes-timex-expression/datatypes_timex_expression/timex_relative_convert.py
from datatypes_timex_expression.english import convert_timex_to_string_relative as english_convert_timex_to_string_relative


class TimexRelativeConvert:

    @staticmethod
    def convert_timex_to_string_relative(timex, reference_date):
        return english_convert_timex_to_string_relative(timex, reference_date)

File: Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexRelativeConvert.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.recognizers.datatypes.timex.expression;

import com.microsoft.recognizers.datatypes.timex.expression.english.TimexRelativeConvertEnglish;

import java.util.Calendar;

public class TimexRelativeConvert {
    public static String convertTimexToStringRelative(TimexProperty timex, Calendar referenceDate) {
        return TimexRelativeConvertEnglish.convertTimexToStringRelative(timex, referenceDate);
    }
}
